MatrixSDK/JSONModels/Login/MXLoginSSOFlow.h

```diff
@@ -31,6 +31,7 @@ extern NSString *const MXLoginSSOFlowIdentityProvidersKey;
  List of all SSO Identity Providers supported
  */
 @property (nonatomic, readonly) NSArray<MXLoginSSOIdentityProvider*> *identityProviders;
+@property (atomic, readonly) BOOL isOIDC;
```

Comment:
Maybe this key should be named to match the JSON `delegatedOIDCCompatibility` or just `delegatedOIDC`